common/journal_entries/00_corn_laws.txt

```diff
@@ -0,0 +1,85 @@
+﻿je_corn_laws = {
+	icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+
+	group = je_group_internal_affairs
+
+	is_shown_when_inactive = {
+		NOT = { has_variable = corn_laws_abolished }
+		ig:ig_landowners = {
+			is_marginal = no
+		}
+		OR = {
+			has_export_priority_tariffs = g:grain
+			has_law = law_type:law_isolationism
+			has_law = law_type:law_mercantilism
+		}
+	}
+
+	possible = {
+		market_capital.market = {
+			mg:grain = { 
+				market_goods_pricier > 0.50 
```

Comment:
I think 0.4 would also be plenty, 0.5 is pretty hard to get, even when deliberately starving your people.